src/pudl/transform/ferc1.py

```diff
@@ -1075,6 +1087,8 @@ def fuel(ferc1_raw_dfs, ferc1_transformed_dfs):
     # (for example) a "Total" line w/ only fuel_mmbtu_per_kwh on it. Grr.
     fuel_ferc1_df.dropna(inplace=True)
 
+    # Replace "unkown" fuel unit with NAs - this comes after we drop missing data with NAs
+    fuel_ferc1_df = fuel_ferc1_df.replace({'fuel_units': 'unknown'}, pd.NA)
```

Comment:
The line above drops all the NAs from the dataframe. I assume we want to keep data that has "unknown" as the fuel type so I replaced the "unknown" fuel types with `pd.NA`s after the `dropna` call. If we want to drop "unknown" fuel types as well then I can just move this `replace` call above `dropna`

Reply:
Wow, that blanket `dropna()` is kind of scary, and probably leftover from very early in the project. I wonder if this is something that @aesharpe has changed in her recent work on the FERC 1 cleaning? This probably requires a closer look at what all is being kept / dropped in the `fuel_ferc1` table during its transformation to see what will be reasonable -- i.e. what criteria do we want to apply for retaining a fuel record -- requiring that it have no NA values seems like it might be excluding too many records.

Reply:
I saw someone (probably Austen) left a new comment above this `dropna()` call but seems like it's staying for the mean time

Reply:
Yeah, it's a different issue really. We should make an issue and re-visit it in other FERC cleaning like you mentioned in your slack message.